edd-blockonomics.php

```diff
@@ -659,6 +659,58 @@ public function settings( $settings )
 </script>
 ';
 
+    $advanced_settings = '<p id="advanced_title" class="bnomics-options-bold"><a href="javascript:show_advanced()">'.__('Advanced Settings', 'edd-blockonomics').'&#9660;</a></p>
+      <div id="advanced_window" style="display:none">
+        <p class="bnomics-options-bold"><a href="javascript:show_basic()">'.__('Advanced Settings', 'blockonomics-bitcoin-payments').' &#9650;</a></p>
+        <table class="form-table">
```

Comment:
Font weight is incorrect on select inputs due to incorrect DOM structure/nested tables. We should use the existing table structure as follows
```
<tr>
	<th scope="row">
		<p id="advanced_title_closed" style="display: none;">
			<a href="javascript:show_advanced()">Advanced Settings ▼</a>
		</p>
		<p class="advanced_title_open" style="display: block;">
			<a href="javascript:show_basic()">Advanced Settings ▲</a>
		</p>
	</th>
</tr>
<tr class="edd-blockonomics-advanced">
	<th scope="row">Time period of countdown timer on payment page (in minutes)</th>
	<td>
		<select id="edd_settings[edd_blockonomics_payment_countdown_time]" name="edd_settings[edd_blockonomics_payment_countdown_time]" data-placeholder="">
			<option value="10" selected="selected">10</option>
			<option value="15">15</option>
			<option value="20">20</option>
			<option value="25">25</option>
			<option value="30">30</option>
		</select>
		<label for="edd_settings[edd_blockonomics_payment_countdown_time]"> </label>
	</td>
</tr>
<tr class="edd-blockonomics-advanced">
	<th scope="row">Network Confirmations required for payment to complete</th>
	<td>
		<select id="edd_settings[edd_blockonomics_confirmations]" name="edd_settings[edd_blockonomics_confirmations]" data-placeholder="">
			<option value="2" selected="selected">2 (recommended)</option>
			<option value="1">1</option><option value="zero">0</option>
		</select>
		<label for="edd_settings[edd_blockonomics_confirmations]"> </label>
	</td>
</tr>
```
We can then simply hide/show the settings by targeting `'class' => 'edd-blockonomics-advanced'`, instead of `'field_class' => 'edd-blockonomics-advanced'`.

Reply:
Okay. Let me restructure that and give it a try.

Reply:
New structure after making the changes

![image](https://user-images.githubusercontent.com/22245433/149055828-575f6239-837d-434c-bba3-266e36b63be0.png)


How it looks:
![image](https://user-images.githubusercontent.com/22245433/149055921-0918f754-1e57-44ae-9861-36130914fbe8.png)

![image](https://user-images.githubusercontent.com/22245433/149056062-7071a019-783f-4ec5-9a35-535d2a32c42b.png)